src/Aspire.Hosting.SurrealDb/SurrealDbContainerImageTags.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+internal sealed class SurrealDbContainerImageTags
+{
+    public const string Image = "surrealdb/surrealdb";
+    public const string Tag = "latest";
```

Comment:
Using `latest` tag bring unexpected behavior. We use `major.minor` tag in most container resources.

Reply:
I am curious about this. That will requires to constantly update this value in the future. Is there any specifics around this? Like a dependabot that update tag version automatically? Shouldn't the version be held somewhere else, like a config project file or a yaml file?

Reply:
Aspire team update tags manually.

Reply:
Ok then.